templates/easyappointments/0/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '2'
+
+services:
+  easyappointments:
+    image: codejamninja/easyappointments:${version}
+{{- if (.Values.public_port)}}
+    ports:
+      - ${public_port}:8888
+{{- end}}
+{{- if not (.Values.mysql_host)}}
+    links:
+      - mysql:db
+{{- end }}
+    environment:
+      BASE_URL: ${base_url}
+      DB_NAME: ${mysql_database}
+      DB_PASSWORD: ${mysql_password}
+      DB_USERNAME: ${mysql_user}
+{{- if (.Values.mysql_host)}}
+      DB_HOST: ${mysql_host}
+{{- end}}
```

Comment:
What is the default value of env var `DB_HOST`?? What do you think to define like that??

```
{{- if not (.Values.mysql_host)}}
    links:
      - mysql:db
+{{- else }}
+    external_links:
+      - ${mysql_host}:db
{{- end }}
    environment:
      BASE_URL: ${base_url}
      DB_NAME: ${mysql_database}
      DB_PASSWORD: ${mysql_password}
      DB_USERNAME: ${mysql_user}
+      DB_HOST: db
-{{- if (.Values.mysql_host)}}
-      DB_HOST: ${mysql_host}
-{{- end}}
```

Reply:
That is so they can connect with a remote database if they so choose to. Read the description of the `msyql_host` question in the rancher config. Also, the default value of DB_HOST is db, hence the reason I link to db when mysql_host is not set.

Reply:
Yeah, i know what you are doing here and it works. Just a proposal that may be more clear.

Reply:
Ah, very nice